exporter/loadbalancingexporter/trace_exporter_test.go

```diff
@@ -581,6 +581,16 @@ func simpleConfig() *Config {
 	}
 }
 
+func attrBasedRoutingConfig() *Config {
+	return &Config{
+		Resolver: ResolverSettings{
+			Static: &StaticResolver{Hostnames: []string{"endpoint-1", "endpoint-2", "endpoint-3"}},
+		},
+		RoutingKey:   "resource",
+		ResourceKeys: []string{"service.name"},
```

Comment:
Can you use a different resource key? `service.name` can be accomplished today already with a routing key of `service`.

Reply:
One more request: could you add another resource key, so that we ensure that multiple resource keys with different values would get routed to different endpoints?